### PR TITLE
enforce storybook v6.2.1

### LINKED ISSUE #863: enforce Storybook v6.2.1

### PR DESCRIPTION
**Pull request by jasonbasuil**
closes #863

### CODE REVIEW

File: spec/__snapshots__/Storyshots.test.js.snap [hunk 1]
```diff
@@ -21208,7 +21208,7 @@ exports[`Storyshots Components/Tabs Controlled 1`] = `
 >
   <div>
     <span
-      className="sc-dmyDGi goryrA"
+      className="sc-dmqHEX hCuTMq"
```
Comment (jasonbasuil) [inline spec/__snapshots__/Storyshots.test.js.snap:21211]: might've been some downstream dependency changes... probably to styled-components or another library which changed the snapshots from the [original bugsnag PR](https://github.com/user-interviews/ui-design-system/pull/845/files#diff-d0b8495a07015b3459796609ae72cddb42e598b9cc3640412c5fb75a9c66c0f7)

Comment (jasonbasuil) [reply spec/__snapshots__/Storyshots.test.js.snap:21211]: was curious about why that was... but this seems to have reverted it back

Comment (rroppolo) [reply spec/__snapshots__/Storyshots.test.js.snap:21211]: Looking at the yarn.lock I think styled-components may have upgraded from 5.3.8 to 5.3.9. Should be fine (unless they pulled a storybook 🙄 )

**Review by rroppolo — APPROVED** [review]
Thanks!!

**Pull request merged**